src/Components/Footer/Social.js

```diff
@@ -45,6 +45,16 @@ function Social() {
             source={require('./../../assets/linkedin-logo.png')}
           />
         </TouchableHighlight>
+        <TouchableHighlight
+          onPress={() =>
+            Linking.openURL('https://medium.com/anitab-org-open-source')
```

Comment:
Great Work @shades-7 but please can you make link to open in new tab so that user won't get redirected from the main website.

Reply:
@nlok5923 That is actually a nice idea, Thanks for the suggestion, but I have tried to maintain the codebase as all the previous social links (Facebook, twitter, etc) were opening in the same tab.
if it's compulsory I will make the changes.

Reply:
Yes the other links aren't that way and are redirecting in same tab only. But can you change all the links to open in new tab? That will be better to redirect links in new tab. Otherwise looks good 👍

Reply:
@keshakaneria so should I create a new issue, then work on that cause the current one is different?

Reply:
Yes you can create but that shall be given to new comers since its a very basic change so that will help new comers to work with Open source Projects.

Reply:
so I think we can merge this issue and create a new one for the redirection of social links?

Reply:
Yes for new comers you can create that issue.